tests/test_submit.py

```diff
@@ -261,24 +266,24 @@ def test_cannot_submit_same_bad_answer_twice(aocd_data_dir, pook, capsys, freeze
     ]
 
 
-def test_will_not_submit_null():
-    with pytest.raises(AocdError("cowardly refusing to submit non-answer: None")):
+def test_will_not_submit_null() -> None:
+    with pytest.raises(AocdError("cowardly refusing to submit non-answer: None")): # type: ignore[call-overload] # using pytest-raisin
         submit(None, part="a")
 
 
 @pytest.mark.answer_not_cached(rv="value")
-def test_submit_guess_against_saved(pook, capsys):
+def test_submit_guess_against_saved(pook: pook_mod) -> None:
     post = pook.post(url="https://adventofcode.com/2018/day/1/answer")
     submit(1234, part="a", day=1, year=2018, session="whatever", reopen=False)
     assert post.calls == 0
 
 
-def test_submit_float_warns(pook, capsys, caplog):
+def test_submit_float_warns(pook: pook_mod, caplog: pytest.LogCaptureFixture) -> None:
```

Comment:
@cj81499  I don't understand why pook fixture (which happens to also yield the pook module) is annotated as the module itself. Why shouldn't it be annotated as `types.ModuleType`?